### DISCUSSION TITLE
How to setup DNS alias to tesla.example.com

### DISCUSSION THREAD

**Discussion by IanK6449** [llamafilm/tesla-http-proxy-addon#38] [General]
Hi,

I am probably showing my ignorance here but in the instructions it states as a prereq:

"You must create an additional DNS record that resolves to the same IP as Home Assistant. For example, if Home Assistant is home.example.com then create tesla.example.com as an alias pointing to the same place."

Do I need to do this in my router or somewhere else?
I ask as I have a Netgear Nighthawk router and I can't see any config options in its admin portal to actually add a DNS alias entry.

Any and all assistant gratefully received and thank you for developing this component to allow us to integrate with our newer Teslas in HASS!

IK

**Comment by jesperholstjh** [comment]
You need to create the DNS record on your domain registrar or wherever your DNS is managed, not on your router. The router typically handles local network settings but not DNS records for your domain.

The reason you need a DNS record that resolves to the same IP as Home Assistant is to ensure that when you access tesla.example.com, it directs you to your Home Assistant instance, home.example.com. This is crucial for proper routing and functionality, especially when integrating external services like Tesla with Home Assistant.

Domains are essentially human-readable addresses for IP addresses, making it easier to access services on the internet. By setting up the correct DNS records, you're ensuring that the necessary services can find and communicate with your Home Assistant setup correctly.
